src/core/figures.js

```diff
@@ -76,7 +76,12 @@ export function run(conf, doc, cb) {
     id = id.substring(1);
     if (figMap[id]) {
       $a.addClass("fig-ref");
-      if ($a.html() === "") $a.append(figMap[id].clone());
+      if ($a.html() === "") {
+        const $shortFigDescriptor = figMap[id].slice(0, 2).clone();
+        const longFigDescriptor = figMap[id].slice(2).clone().text();
+        $a.attr("title", longFigDescriptor.trim());
```

Comment:
Small edge case here, but we should only set `title`, if the `title` attribute does not exist.

Reply:
Oops! Didn't think of that!